src/audio/audio_stream.c

```diff
@@ -49,7 +49,8 @@ static size_t audio_stream_get_data_available(struct sof_source *source)
 }
```

Comment:
@marcinszkudlinski Maybe remove the "Conflicts: .." bits from commit message...?